libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -132,7 +133,16 @@ string IRGenerator::generateFunction(FunctionDefinition const& _function)
 	string functionName = m_context.functionName(_function);
 	return m_context.functionCollector().createFunction(functionName, [&]() {
 		Whiskers t(R"(
-			function <functionName>(<params>) <returns> {
+			function <functionName>(<params><?isConstructor><comma><additionalParams></isConstructor>) <returns> {
+				<?isConstructor>
+					<initBaseArguments>
+
+					<?hasNextConstructor>
```

Comment:
Could you just set `nextConstructorCall` to the empty string if there is no next constructor?